content/changelog/2024-04-05-console.md

```diff
@@ -0,0 +1,48 @@
+---
+
+### New Monitoring widget on the Dashboard
+
+We've added a **Monitoring** widget to the Neon **Dashboard**, giving you an easy way to check CPU and RAM usage for your Neon projects. The display defaults to your primary branch but you can switch to view usage details for any branch in your project.
+
+![monitoring widget](/docs/relnotes/monitoring_widget.png "no-border")
+
+We're committed to improving observability for your Neon projects and databases. To learn more about monitoring in Neon, see [Monitoring](/docs/introduction/monitoring).
+
+### A teaser for organizations
+
+You might notice a new addition to our breadcrumb navigation: **Create organization** under your account selector. It's greyed out and inactive for now, but we've included it here as a reminder: organization support is coming soon!
+
+![organization teaser](/docs/relnotes/org_teaser.png "no-border")
+
+Check our [roadmap](/docs/introduction/roadmap) to see what else is coming next.
+
+### PostgreSQL Partition Manager extension
+
+Neon now supports the [pg_partman](/docs/extensions/pg_partman) Postgres extension, which is a partition manager extension that enables creating and managing time and number-based table partition sets in Postgres.
+
+### Local File Cache (LFC) statistics and working set size function
+
+You can now view Local File Cache (LFC) statistics for your Neon compute. The statistics include a cache hit ratio, which can help you determine if your data is accessed from memory or the Neon storage layer.
+The Local File Cache (LFC) stores frequently accessed data in the local memory of your Neon compute instance. The cache helps to reduce latency and improve query performance by minimizing fetches from the Neon storage layer. The LFC acts as an add-on or extension of Postgres [shared buffers](#shared-buffers). In Neon, the `shared_buffers` setting is always 128 MB, regardless of compute size. The LFC extends cache memory up to 80% of your compute's RAM.
+Local File Cache (LFC) statistics are exposed through a `neon_stat_file_cache view`, which you access by installing the `neon` extension.
+
+```sql
+CREATE EXTENSION neon;
+```
+
+You can run the following query to view statistics:
+
+```sql
+SELECT * FROM neon_stat_file_cache;
+ file_cache_misses | file_cache_hits | file_cache_used | file_cache_writes | file_cache_hit_ratio
+-------------------+-----------------+-----------------+-------------------+----------------------
+           2133643 |       108999742 |             607 |          10767410 |                98.08
+```
+
+When the `neon` extension is installed, you can also use `EXPLAIN ANALYZE` with the `FILECACHE` option to view LFC hit and miss data. For more information, see [The neon extension](https://neon.tech/docs/extensions/neon). To learn how the LFC cache hit ratio can help you right-size your compute, see [How to size your compute](/docs/manage/endpoints#how-to-size-your-compute).
```

Comment:
To use the EXPLAIN (ANALYZE, FILECACHE) option it is NOT necessary to install the neon extension.
The explain capability is implemented independently of the extension in https://github.com/neondatabase/neon/pull/6851

Reply:
Thank you. Fixed.